src/Libraries/Microsoft.Extensions.Http.Resilience/Resilience/Internal/StrategyKeyProviderHelper.cs

```diff
@@ -17,25 +18,22 @@ public static void SelectStrategyByAuthority(IServiceCollection services, string
         {
             var redactor = serviceProvider.GetRequiredService<IRedactorProvider>().GetRedactor(classification);
 
-            return new ByAuthorityStrategyKeyProvider(redactor);
+            return new ByAuthorityStrategyKeyProvider(redactor).GetStrategyKey;
```

Comment:
Perhaps rename GetStrategyKey to just StrategyKey?

Reply:
`GetStrategyKey` is just a function that gets converted to a delegate. I think the name is ok here